content/docs/concepts/certificaterequest.md

```diff
@@ -60,16 +60,37 @@ manage the logic and life cycle of `CertificateRequests`.
 used and relied upon by controllers or services to make decisions on what
 actions to take next on the resource.
 
-
 ### Ready
 Each ready condition consists of the pair `Ready` - a boolean value, and
 `Reason` - a string. The set of values and meanings are as follows:
 
 | Ready | Reason  | Condition Meaning                                                                                                                                                                                                                               |
 | ----- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | False | Pending | The `CertificateRequest` is currently pending, waiting for some other operation to take place. This could be that the `Issuer` does not exist yet or the `Issuer` is in the process of issuing a certificate.                                   |
-| False | Failed  | The certificate has failed to be issued - either the returned certificate failed to be decoded or an instance of the referenced issuer used for signing failed. No further action will be taken on the `CertificateRequest` by it's controller. |
-| True  | Issued  | A signed certificate has been successfully issued by the referenced `Issuer`.                                                                                                                                                                   |
+| False | Failed  | The certificate has failed to be issued - either the returned certificate failed to be decoded or an instance of the referenced issuer used for signing failed. No further action will be taken on the `CertificateRequest` by it's controller and it can be considered terminally failed. |
```

Comment:
```suggestion
| False | Failed  | The certificate has failed to be issued - either the returned certificate failed to be decoded or an instance of the referenced issuer used for signing failed. No further action will be taken on the `CertificateRequest` by its controller and it can be considered terminally failed. |
```

Reply:
Thank you @wallrj I keep automatically typing one instead of the other for these two. Should be fixed now.